sdk/python/endpoints/online/llm/langchain/deployments/env.yml

```diff
@@ -0,0 +1,16 @@
+name: plugin-demo-env
```

Comment:
let's remove any naming related to plugin

Reply:
changed to llm-demo-env